workloads/kube-burner-ocp-wrapper/run.sh

```diff
@@ -124,7 +124,6 @@ download_binary
 if [[ ${WORKLOAD} =~ "index" || ${WORKLOAD} =~ "workers-scale" ]]; then
   if [[ "$START_TIME" != 0 && "$END_TIME" != 0 ]]; then
     JOB_START=$(date -u -d "@$START_TIME" +"%Y-%m-%dT%H:%M:%SZ")
-    END_TIME=$(expr $END_TIME + 600)
```

Comment:
You removed this line, but the variable is used below.

Reply:
Still its valid and the variable is present. We are just removing the 10 minutes add at the end.